lib/IDE/CodeCompletion.cpp

```diff
@@ -893,6 +898,17 @@ static void addAnyTypeKeyword(CodeCompletionResultSink &Sink, Type T) {
   Builder.addTypeAnnotation(T, PrintOptions());
 }
 
+static void addClosureSignatureKeywords(CodeCompletionResultSink &Sink,
+                                        DeclContext *DC) {
+  ClosureExpr *closure = dyn_cast<ClosureExpr>(DC);
+  if (!closure)
+    return;
+  if (closure->getInLoc().isValid())
+    return;
+
+  addKeyword(Sink, "in", CodeCompletionKeywordKind::kw_in);
```

Comment:
IIUC we won’t suggest `in` in
```swift
let x = [1,2,3]
for y #^COMPLETE^#
```
anymore. Do we have a test case for that?

Reply:
Oooh, good catch. I don't think we currently suggest `in` for `for element #^HERE^#` though. 
```
func test(inValues: [Int]) {
  for value #^HERE^# 
}
```
suggests `inValues`, but not `in`. I think this hasn't been an issue because users usually just hit `in` and space bar, which doesn't select anything from the completion UI.

Reply:
Oh, interesting. I filed https://github.com/apple/swift/issues/58633 to keep track of that issue. If it was broken before, I’m fine with keeping it as-is.